mappings/net/minecraft/block/Block.mapping

```diff
@@ -318,7 +319,7 @@ CLASS net/minecraft/class_2248 net/minecraft/block/Block
 		ARG 1 state
 		ARG 2 mirror
 	METHOD method_9570 hasBlockEntity ()Z
-	METHOD method_9571 (Lnet/minecraft/class_2680;Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;)Lnet/minecraft/class_265;
+	METHOD method_9571 getCullShape (Lnet/minecraft/class_2680;Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;)Lnet/minecraft/class_265;
```

Comment:
This is also mapped in #839 as `getCullingShape`.

Reply:
so is cull shape or culling shape better

Reply:
should no longer conflict with @Sollace 's pr.